spec/models/miq_server_spec.rb

```diff
@@ -79,14 +79,6 @@
       end
     end
 
-    it "should have proper guid" do
-      expect(@miq_server.guid).to eq(@guid)
-    end
-
-    it "should have default zone" do
-      expect(@miq_server.zone.name).to eq(@zone.name)
-    end
```

Comment:
Why were these dropped?

Reply:
If you look at the code

```ruby
  def self.create_guid_miq_server_zone
    server = local_miq_server
    [server.guid, server, server.zone]
  end

  @guid, @miq_server, @zone = EvmSpecHelper.create_guid_miq_server_zone

    it "should have proper guid" do
      expect(@miq_server.guid).to eq(@guid)
    end

    it "should have default zone" do
      expect(@miq_server.zone.name).to eq(@zone.name)
    end
```

So the test is essentially:
```ruby
      @guid = @miq_server.guid
      expect(@miq_server.guid).to eq(@guid)
```

Which felt useless to me.

Reply:
Yeah it's a test for create_guid_miq_server_zone itself, which you are dropping.  I agree :+1: